packages/browser-utils/src/metrics/browserMetrics.ts

```diff
@@ -129,12 +129,12 @@ export function startTrackingLongAnimationFrames(): void {
   // we directly observe `long-animation-frame` events instead of through the web-vitals
   // `observe` helper function.
   const observer = new PerformanceObserver(list => {
+    if (!getActiveSpan()) {
+      return;
+    }
     for (const entry of list.getEntries() as PerformanceLongAnimationFrameTiming[]) {
-      if (!getActiveSpan()) {
-        return;
-      }
       if (!entry.scripts[0]) {
-        return;
+        break;
```

Comment:
This is a behaviour change. Would you mind explaining why we should `break` here?

Reply:
Thank you for pointing that out. It was a typo, and I have corrected it to `continue`. The change from `return` to `continue` is to ensure that the remaining `entry` containing `initialScript` in `list.getEntries()` are not skipped and are reported properly.